joern-cli/frontends/pysrc2cpg/src/main/scala/io/joern/pysrc2cpg/PythonTypeHintCallLinker.scala

```diff
@@ -18,4 +22,47 @@ class PythonTypeHintCallLinker(cpg: Cpg) extends XTypeHintCallLinker(cpg) {
     case typ => typ
   }
 
+  override def linkCalls(builder: DiffGraphBuilder): Unit = {
+    val methodMap = mutable.HashMap.empty[String, MethodBase]
+    val callerAndCallees = calls
+      .map(call => (call, calleeNames(call)))
+      .toList
+    // Gather all method nodes and/or stubs
+    callerAndCallees
+      .foreach { case (call, methodNames) =>
+        val ms = callees(methodNames).l
+        if (ms.nonEmpty) {
+          ms.foreach { m => methodMap.put(m.fullName, m) }
+        } else {
+          val mNames = ms.map(_.fullName).toSet
+          methodNames
+            .filterNot(mNames.contains)
+            .map(fullName => createMethodStub(fullName, call, builder))
+            .foreach { m => methodMap.put(m.fullName, m) }
+        }
+      }
+    // Link edges to method nodes
+    callerAndCallees.foreach { case (call, methodNames) =>
+      methodNames
+        .flatMap(methodMap.get)
+        .foreach { m => builder.addEdge(call, m, EdgeTypes.CALL) }
+      val methodNamesFiltered = methodNames.filter( m => !(m.contains(DummyReturnType) &&
```

Comment:
`XTypeRecovery` on `origin/master` contains a call `isDummyType` that checks this.

Reply:
Changes in this file are no longer needed after [this](https://github.com/joernio/joern/blob/437c4a73d46ed25bfe549861481ac9c6bf05e929/joern-cli/frontends/pysrc2cpg/src/main/scala/io/joern/pysrc2cpg/PythonAstVisitor.scala#L1690) change. Restored this code to what it is in `master`